pkgs/games/heroic/default.nix

```diff
@@ -1,48 +1,41 @@
 { lib
 , stdenv
 , fetchFromGitHub
-, fetchYarnDeps
-, yarn
-, fixup-yarn-lock
+, pnpm
 , nodejs
-, python3
 , makeWrapper
 , electron
 , vulkan-helper
 , gogdl
 , legendary-gl
 , nile
+, comet-gog
 }:
 
 let appName = "heroic";
-in stdenv.mkDerivation rec {
+in stdenv.mkDerivation (finalAttrs: {
   pname = "heroic-unwrapped";
-  version = "2.14.1";
+  version = "2.15.1";
 
   src = fetchFromGitHub {
     owner = "Heroic-Games-Launcher";
     repo = "HeroicGamesLauncher";
-    rev = "v${version}";
-    hash = "sha256-AnyltqNP+XyVzgCobM3g6DIXntD3spKecYtCRx+8oic=";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-+OQRcBOf9Y34DD7FOp/3SO05mREG6or/HPiOkasHWPM=";
   };
 
-  offlineCache = fetchYarnDeps {
-    yarnLock = "${src}/yarn.lock";
-    hash = "sha256-3CYSw1Qy363eyhy3UyFgihSau+miNHwvKjhlq/kWxWQ=";
+  pnpmDeps = pnpm.fetchDeps {
+    inherit (finalAttrs) pname version src;
+    hash = "sha256-3PiB8CT7wxGmvRuQQ5FIAmBqBm9+R55ry+N/qUYWzuk=";
   };
 
   nativeBuildInputs = [
-    yarn
-    fixup-yarn-lock
     nodejs
-    python3
+    pnpm.configHook
     makeWrapper
   ];
 
   patches = [
-    # Reverts part of upstream PR 2761 so that we don't have to use a non-free Electron fork.
-    # https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/2761
-    ./remove-drm-support.patch
```

Comment:
Please drop the patch from the source tree

Reply:
Oops!